lib/Elastica/Client.php

```diff
@@ -640,6 +647,14 @@ public function request($path, $method = Request::GET, $data = array(), array $q
 
             return $this->request($path, $method, $data, $query);
         }
+
+        $this->_logger->debug('Elastica Request', [
```

Comment:
Just a thought: I'm not sure how the implementation of Elastica\Log will handle the context array if we have any unserializable object as part of the Request (from what it looks like, the Response would die before we get here if it had anything unserializable).